src/components/Header/Header.css

```diff
@@ -1,27 +1,23 @@
 .Header {
-    @apply
-      w-full
-      bg-white;
+  @apply w-full
```

Comment:
I think I did it because the spacing value scale jumps from `20` -> `30` to keep things standard. Even though the system is multiple of 4s it's still good to constrain to a discrete set of values instead of a full range as this constraint forces designer and developers to stick to a standard and not just use whatever value is convenient or "feels" right